dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -8,6 +8,9 @@ RUN apt-get install -f -y ifupdown bridge-utils libdbus-1-3 libdaemon0 libjansso
 ## TODO: implicitly install dependencies
 RUN apt-get -y install libjemalloc1
 
+# Dependencies for sonic-cfggen                                                                                                                                                           
+RUN apt-get install -y python-lxml python-jinja2 python-netaddr python-ipaddr python-yaml 
```

Comment:
Maybe it is reasonable to move installation of those packages to docker-base image. List of packages are common for all containers. This will reduce space usage.